engine/engine/content/builtins/manifests/android/AndroidManifest.xml

```diff
@@ -25,6 +25,7 @@
                 android:configChanges="fontScale|keyboard|keyboardHidden|locale|mcc|mnc|navigation|orientation|screenLayout|screenSize|smallestScreenSize|touchscreen|uiMode"
                 android:theme="@android:style/Theme.NoTitleBar.Fullscreen"
                 android:screenOrientation="{{orientation-support}}"
+                android:exported="true"
```

Comment:
This is a new requirement for Android 12:
If your app targets Android 12 or higher and contains [activities](https://developer.android.com/guide/components/activities/intro-activities), [services](https://developer.android.com/guide/components/services), or [broadcast receivers](https://developer.android.com/guide/components/broadcasts) that use [intent filters](https://developer.android.com/guide/components/intents-filters#Receiving), you must explicitly declare the [android:exported](https://developer.android.com/guide/topics/manifest/activity-element#exported) attribute for these app components.

https://developer.android.com/about/versions/12/behavior-changes-12#exported